src/app.ts

```diff
@@ -184,4 +185,21 @@ angular.module('3ema', [
     }]);
 }])
 
+.factory('$exceptionHandler', ['LogService', function(logService: LogService) {
+    const logger = logService.getLogger('UncaughtException');
+    return function myExceptionHandler(exception: Error, cause) {
+        const data: any[] = [];
+        const message = exception.message.includes(exception.name)
+            ? exception.message
+            : `${exception.name}: ${exception.message}`;
```

Comment:
Aaaaah I can't do that because it's possible to throw non-`Error` types :roll_eyes: